src/client/src/App/Analytics/Positions/__tests__/Positions.test.tsx

```diff
@@ -1,11 +1,10 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
+import { BehaviorSubject } from "rxjs"
```

Comment:
import re-ordering, per offline disc .. we need to use a tool - vscode built-in is a bit basic and also won't run off a script ... suggest `eslint-plugin-import`, and then all commits can be `yarn verify`'ed

Reply:
@algreasley - You mean https://github.com/lydell/eslint-plugin-simple-import-sort/ ?

Reply:
The plugin I'm using requires less configuration with TypeScript, and seems much simpler overall.  Can be integrated with VSCode too.